server/server.py

```
@@ -0,0 +1,223 @@
+        status_message = "OK" if status_code in (200, 201) else "Error"
+        response = (f"HTTP/1.1 {status_code} {status_message}\r\n"
+                    f"Content-Type: application/json\r\n"
+                    f"Content-Length: {len(response_body_json)}\r\n\r\n"
+                    f"{response_body_json}")
+        client_socket.sendall(response.encode())
+    except Exception as e:
+        error_response = json.dumps({"error": "Server Error", "message": str(e)})
+        response = f"HTTP/1.1 500 Internal Server Error\r\nContent-Type: application/json\r\nContent-Length: {len(error_response)}\r\n\r\n{error_response}"
+        try:
+            client_socket.sendall(response.encode())
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)